themes/default/content/blog/build-images-50x-faster-docker-v4/index.md

```diff
@@ -0,0 +1,147 @@
+    - providers
+
+---
+
+The Pulumi Docker Provider has been a top Pulumi provider since it launched in 2018. It can be used to provision any of the resources available in Docker, including managing containers, images, networks, volumes and more.
+
+One of the most heavily used features of this provider is the `docker.Image` resource, which enables Pulumi users to build and (optionally) push a local Docker context (like an application folder), to a registry as part of a Pulumi deployment. Today we are excited to announce a set of improvements to the Pulumi Docker Provider Image Resource driven by the feedback we have received from our community. This set of improvements includes:
+
+* Significantly improved performance (including reduced need for rebuilds)
+* BuildKit support (including cross-platform builds)
+* Rich Docker build logs inside Pulumi IaC
+* Pulumi YAML and Pulumi Java support
+
+<!--more-->
+
+### Significantly improved performance
+
+To improve the efficiency of the Docker Provider, we have changed the default behavior to rebuild only on context change. In addition, the `cacheFrom` option now offers the ability to use a registry as the image cache. By leveraging BuildKit and caching improvements, subsequent rebuilds may be up to 50x faster. This functionality and other improvements of the Docker Provider we are announcing today are [highly requested from our community](https://github.com/pulumi/pulumi-docker/issues/132).
+
+<!-- Add image/gif showing output during build, side by side comparison of old vs new?, or graph with speed comparison -->
+
+### BuildKit support
+
+The enhanced functionality and performance provided by the [Docker BuildKit](https://docs.docker.com/build/buildkit/) is now fully available to be leveraged in your Pulumi programs. BuildKit comes with functionality to improve your builds’ performance and the reusability of your Dockerfiles. Cross-platform builds are now supported, enabling image building for a target container runtime that is different from the build environment. BuildKit is now the default builder version mode. The V1 builder can still be accessed by specifying `Build.BuilderVersion=BuilderV1`.
+
+```typescript
+import * as docker from "@pulumi/docker";
+const img = new docker.Image("my-image", {
+    imageName: "docker.io/pulumibot/demo-image:latest",
+    build: {
+        context: "app",
+        dockerfile: "Dockerfile",
+        args: {
+            "BUILDKIT_INLINE_CACHE": "1"
+        },
+        builderVersion: "BuilderBuildKit", // can also be set to `BuilderV1`
+        cacheFrom: {
+            images: ["docker.io/pulumibot/demo-image:cache-base"]
+        },
+        platform: "linux/amd64",
+    }
+});
+```
+
+## Rich Docker build logs inside Pulumi IaC
+
+Filtered docker logs during build and push will now be displayed in the Info box on pulumi up.
+[![asciicast](https://asciinema.org/a/I8Xzmfme56ZP4uD6uo2U4i2wr.svg)](https://asciinema.org/a/I8Xzmfme56ZP4uD6uo2U4i2wr)
```

Comment:
i recommend downloading the GIF file and including it in Hugo so it can be embedded in here and not need to link off to a 3rd party site